src/headerData.json

```diff
@@ -180,8 +180,8 @@
     "type": "blocker",
     "message": "Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.",
     "button": {
-      "action": "validateSubscribersPopup",
-      "text": "Verificar Ahora"
+      "action": "updatePlanPopup",
```

Comment:
Take in account that header type `blocker` is a red one.  If you want the one for update please check the type, `info` for green.